### DISCUSSION TITLE
How do i enable AD refill?

### DISCUSSION THREAD

**Discussion by talianagisan** [Courseplay/Courseplay_FS22#1191] [Q&A]
So my field is very big and I need to lime it. How do I set it up so when CP is empty it engages autodrive and refills itself then goes back to the field and continues where it left off? The features tab says it can do this as refill and unloaded handled by autodrive is listed.

**Comment by GeorgePikoulis** [comment]
@talianagisan I think we need to wait for the next version of Autodrive, 2.0.0.5, while at the moment it's at 2.0.0.4.

**Reply by Tensuko** [reply]
AD can do this already with 2.0.0.4
You have to enable to CP Button on the AD HUD.
There is not setting anymore on CP.
It is like AD watches CP Driver on or off.
Ofc you got to set the right mode for AD.

**Reply by talianagisan** [reply]
> AD can do this already with 2.0.0.4 You have to enable to CP Button on the AD HUD. There is not setting anymore on CP. It is like AD watches CP Driver on or off. Ofc you got to set the right mode for AD.

I see. It'll take awhile to relearn but this'll help. Makes getting my 70kL of lime much much easier.  While I have you, articulated tractors wiggle reversing like they got hips that don't lie. Is this a know bug or should I make a formal report when I have time?

**Reply by Tensuko** [reply]
When CP drive them or AD ?

**Reply by talianagisan** [reply]
> When CP drive them or AD ?

Course play. While reversing to line up it wiggles back and forth lightly at high frequency and goes really slowly. All my tests have been drawbar or pintle or trailered not solid mount

**Reply by Tensuko** [reply]
Then you can make a bug report and use the template pls... Reak Wiki that helps too.
